test/unit/network_spec.js

```diff
@@ -94,7 +94,7 @@ describe('network', function() {
         rangeChunkSize: 65536,
         disableStream: false,
       },
-      disableRange: false
+      disableRange: true
```

Comment:
Please note that in the updated patch, there's no functional changes.
However, in order to keep the same general structure of all the tests, I've added some currently missing `expect` statements, and the same Promise handling throughout, rather than removing code. I hope this is deemed acceptable!